common/message.ts

```diff
@@ -24,26 +29,24 @@ export const enum TelemetryInfoType {
   VPD = 'vpd',
 }
 
-export const enum ResponseErrorInfoMessage {
+export const enum ResponseErrorInfoMessage { // not sure if each word should be connected with '-'
   InvalidRequestType = 'Invalid or missing request type.',
   InvalidTelemetryInfoType = 'The requested telemetry infoType is either invalid or missing.',
+  InvalidDiagnosticsRoutineStatus = 'The diagnostics routine status is invalid.',
   MissingTelemetryRequest = 'Missing telemetry object in request.',
   MissingDiagnosticsRequest = 'Missing diagnostics object in request.',
   InvalidDiagnosticsAction = 'The requested diagnostics action is either invalid or missing.',
+  InvalidDiagnosticsParams = 'The requested diagnostics params are either invalid or missing.',  
   InvalidDiagnosticsRoutineName = 'The requested diagnostics routine name is either invalid or missing.',
   InvalidDiagnosticsRoutineId = 'The requested diagnostics routine id is either invalid or missing.',
   UnsupportedTelemetryFunction = 'Unsupported operation. The requested telemetry function is not supported',
 }
 
-export const enum DiagnosticsRoutineName {
-  RUN_BATTERY_CAPACITY_ROUTINE = 'runBatteryCapacityRoutine',
-}
-
 export const enum DiagnosticsAction {
   START = 'start',
   STATUS = 'status',
-  RESUME = 'resume',
   STOP = 'stop',
+  RESUME = 'resume',
```

Comment:
Alphabetical order, lets move RESUME to the top

Reply:
Done.